homework9/src/components/Burger/Burger.js

```diff
@@ -0,0 +1,20 @@
+import clsx from "clsx";
+import React from "react";
+
+import "./Burger.module.css";
+
+function Burger(props) {
+  return (
+    <div
+      className={clsx("burger", !props.open && "change")}
```

Comment:
second sollution: `className={clsx("burger", {"change": !props.open} )}` just for your information